apps/root/src/pages/earn/components/all-strategies-table/index.tsx

```diff
@@ -192,30 +203,53 @@ const AllStrategiesTable = () => {
     void fetchStrategies();
   }, []);
 
+  const handleSearchChange = (newValue: string) => {
+    setPage(0);
+    setSearch(newValue);
+  };
+
+  const filteredStrategies = React.useMemo<Strategy[]>(
```

Comment:
btw remember that search WILL trigger a call to the sdk to fetch more vaults probably, maybe we should work on the filters + pagination first, so we can have that already done and then when we implement the search is a bit more straightforward

Reply:
Agree, will resume this next week then!